### ISSUE TITLE
caldera_forms_entry_viewer showing "No Entries To Display" when there is 4 records

### ISSUE THREAD

**Issue by ngugliotti** [CalderaWP/Caldera-Forms#1188]
Caldera Forms = Version 1.5.0-b-2
WordPress = Version 4.7.2
PHP = Version 5.4.45

I am trying to use the [caldera_forms_entry_viewer id="CF588e1a006c17a"] option on a post and or page, but I can not get it to display the 4 entries I have saved.

Any ideas on where I can start to troubleshoot? I downloaded the lastest version and tried a bunch of obvious things but nothing seems to be working.

Thanks in advance for your help!

**Comment by andrewlimaza** [comment]
I have just tested this and unable to recreate this issue on my local environment.

Caldera Forms = Version 1.5.0-b-2
WordPress = Version 4.7.2
PHP = Version 5.5.24

>  I downloaded the lastest version and tried a bunch of obvious things but nothing seems to be working.

Could you please explain what other issues you are experiencing?

There may be an underlying issue on your website, would you mind having a look at the following blog post to debug in WordPress - https://calderaforms.com/2016/05/wordpress-debug-logging/

Please let me know if there are any errors you may find.

**Comment by ngugliotti** [comment]
Andrew, Thank you very much for the help!

I turned on debug mode, and here is what it said:

![image](https://cloud.githubusercontent.com/assets/7011168/22510684/d84dd47e-e860-11e6-8ac5-d2f7f3eba60e.png)

**Comment by Shelob9** [comment]
@ngugliotti Thanks for turning on debug mode. I bet those same errors are effecting the API request to get the entries. Can you try again with the plugins that are causing those issues disable?

**Comment by ngugliotti** [comment]
ok so I switched to a wordpress (2015) theme and currently have no errors showing up with debug mode on, but I am still getting the same results.

![image](https://cloud.githubusercontent.com/assets/7011168/22524586/d56e0680-e891-11e6-92c9-7beb1dd190d1.png)

**Comment by Shelob9** [comment]
Are you logged in as an admin user? If not this is the correct behavior, based on your shortcode.

**Comment by ngugliotti** [comment]
Yes I am logged in as Admin, how can I get it to display for all users?

But most importantly its not displaying for admin

**Comment by ngugliotti** [comment]
I noticed that I have 2 sections for each form I have, this doesnt seem right

![image](https://cloud.githubusercontent.com/assets/7011168/22525583/bb4fff3e-e895-11e6-90b4-2507f8c006ab.png)

**Comment by Shelob9** [comment]
Thanks for the additional info. I'll look into this.

The double forms in the list is a known issue with the 1.5 beta.

I appreciate you testing the beta, but keep in mind it is not done yet. We are still running the current release version 1.4.8 on our own site.

**Comment by ngugliotti** [comment]
Thank you for the support, if I need to purchase "Priority Support" just let me know, thanks

**Comment by Shelob9** [comment]
I strongly encourage you to buy an add-on which comes with support, it helps support development of cool new features like this.
